src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -118,6 +118,30 @@ class TimeSeriesRDD(val index: DateTimeIndex, parent: RDD[(String, Vector[Double
   }
 
   /**
+   * Takes the first N series in the TimeSeriesRDD and returns a new, reduced, TimeSeriesRDD.
+   * Works by creating an array of series to include and then checks for membership.
+   * @param n the number of series to take (first n taken) n >= 0
+   * @return reduced TimeSeriesRDD
+   */
+  def takeNSeries(n: Int): TimeSeriesRDD = {
```

Comment:
In Spark, `take` collects data to the driver, so a method with a similar name that returns an RDD might surprise people.  Is this method needed for something else you're working on?